packages/guides/integrating-the-safe-core-sdk.md

```diff
@@ -0,0 +1,353 @@
+
+To integrate the [Safe Core SDK](https://github.com/gnosis/safe-core-sdk) into your Dapp or script you would need to install these dependencies:
+
+```
+@gnosis.pm/safe-core-sdk-types
+@gnosis.pm/safe-core-sdk
+@gnosis.pm/safe-service-client
+```
+
+## 2. Initialize the SDK’s <a name="initialize-sdks"></a>
+
+### Initialize the Safe Service Client
+
+As stated in the introduction, the [Safe Service Client](https://github.com/gnosis/safe-core-sdk/tree/main/packages/safe-service-client) consumes the [Safe Transaction Service API](https://github.com/gnosis/safe-transaction-service). To start using this library create a new instance of the class `SafeServiceClient` imported from `@gnosis.pm/safe-service-client` and pass to its constructor the URL of the Safe Transaction Service you want to use depending on the network.
+
+```js
+import SafeServiceClient from '@gnosis.pm/safe-service-client'
+
+const transactionServiceUrl = 'https://safe-transaction.gnosis.io'
+const safeService = new SafeServiceClient(transactionServiceUrl)
+```
+
+### Initialize the Safe Core SDK
+
+The [Safe Core SDK](https://github.com/gnosis/safe-core-sdk/tree/main/packages/safe-core-sdk) library only interacts with the [Safe contracts](https://github.com/gnosis/safe-contracts). Because of that, we need to select one Ethereum library between [web3.js](https://web3js.readthedocs.io/) and [ethers.js](https://docs.ethers.io/v5/) to interact with the blockchain.
+
+* **Using ethers.js**
+
+  We can use the class `EthersAdapter` from `@gnosis.pm/safe-core-sdk` as a wrapper of `ethers.js`.
+
+  ```js
+  import { EthersAdapter } from '@gnosis.pm/safe-core-sdk'
+  import { ethers } from 'ethers'
+
+  const web3Provider = // ...
+  const provider = new ethers.providers.Web3Provider(web3Provider)
+  const safeOwner = provider.getSigner(0)
+
+  const ethAdapter = new EthersAdapter({
+    ethers,
+    signer: safeOwner
+  })
+  ```
+
+* **Using web3.js**
+
+  We can use the class `Web3Adapter` from `@gnosis.pm/safe-core-sdk` as a wrapper of `web3.js`.
+
+  ```js
+  import { Web3Adapter } from '@gnosis.pm/safe-core-sdk'
+  import Web3 from 'web3'
+
+  const ethAdapter = new Web3Adapter({
+    web3,
+    signerAddress: safeOwnerAddress
+  })
+  ```
+
+Once we have an instance of `EthersAdapter` or `Web3Adapter` we are ready to instantiate the `SafeFactory` and `Safe` classes from `@gnosis.pm/safe-core-sdk`.
+
+```js
+import Safe, { SafeFactory } from '@gnosis.pm/safe-core-sdk'
+
+const safeFactory = await SafeFactory.create({ ethAdapter })
+
+const safeSdk = await Safe.create({ ethAdapter, safeAddress })
+```
+
+There are two versions of the Safe contracts: [GnosisSafe.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafe.sol) that does not trigger events in order to afford some gas and [GnosisSafeL2.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafeL2.sol) that does it and it is more appropriate for L2 networks.
```

Comment:
"...in order to save gas..."

Reply:
Isn't it confusing using "doesn't" if it refers to not triggering the events? (double negative)

Reply:
True, my bad. This would be it then:

There are two versions of the Safe contracts: [GnosisSafe.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafe.sol) that does not trigger events in order to save gas and [GnosisSafeL2.sol](https://github.com/gnosis/safe-contracts/blob/v1.3.0/contracts/GnosisSafeL2.sol) that does, which is more appropriate for L2 networks.